app/config/collections.php

```diff
@@ -1698,6 +1698,39 @@
             ],
         ],
     ],
+    Database::SYSTEM_COLLECTION_REALTIME_CONNECTIONS => [
+        '$collection' => Database::SYSTEM_COLLECTION_COLLECTIONS,
```

Comment:
I think we can just keep the name to `SYSTEM_COLLECTION_CONNECTIONS`, we might be able to reuse it in the future, and it will be more consisted with our 1 level names across all other collections.